src/dom_types/values.rs

```diff
@@ -33,6 +33,35 @@ impl From<&CSSValue> for CSSValue {
     }
 }
 
+// ----------- ToCssValue impls ------------
+
+// impl<T: ToString> ToCssValue for T
+#[doc(hidden)]
+pub trait ToCssValueForToString {
+    fn to_css_value(&self) -> CSSValue;
+}
+
+impl<T: ToString> ToCssValueForToString for T {
+    fn to_css_value(&self) -> CSSValue {
+        CSSValue::Some(self.to_string())
+    }
+}
+
+// impl<T: ToString> ToCssValue for Option<T>
+#[doc(hidden)]
+pub trait ToCssValueForOptionToString {
+    fn to_css_value(&self) -> CSSValue;
+}
+
+impl<T: ToString> ToCssValueForOptionToString for Option<T> {
+    fn to_css_value(&self) -> CSSValue {
+        self.as_ref()
+            .map_or(CSSValue::Ignored, |t| CSSValue::Some(t.to_string()))
+    }
+}
+
+// TODO: Should we impl ToCssValue for more types? such as Vec<T>, Box<T>.. etc
```

Comment:
Implement it for `CSSValue` please (tests fail on `server_integration` example). I don't know if we need other types and we can implement them later.

Run `cargo make start server_integration` or `cargo make verify_only` or `cargo make verify` to test it.
And you should run `cargo make verify` before push to make sure the code is formatted and linted (see [CONTRIBUTING.md](https://github.com/David-OConnor/seed/blob/master/CONTRIBUTING.md)).

Reply:
> Implement it for CSSValue please (tests fail on server_integration example). I don't know if we need other types and we can implement them later.

Oh forget that, Fixed

> Run cargo make start server_integration or cargo make verify_only or cargo make verify to test it.
And you should run cargo make verify before push to make sure the code is formatted and linted (see CONTRIBUTING.md).

I have tried that before .. but was saying verify was not found.. now it works!
it pass all test, but there is a message from clippy:
``` shell
...
...
...

[cargo-make] INFO - Running Task: fmt_all_check
[cargo-make] INFO - Running Task: clippy
[cargo-make] INFO - Execute Command: "cargo" "clippy" "--all-features" "--" "--deny" "warnings" "--deny" "clippy::pedantic" "--deny" "clippy::nursery"
    Checking seed v0.4.1 (/home/muhannad/أعمالي/البرمجة/Rust/seed)
error: this could be a const_fn
   --> src/shortcuts.rs:334:1
    |
334 | / pub fn wrap_debug<T>(object: T) -> dbg::WrapDebug<T> {
335 | |     dbg::WrapDebug(object)
336 | | }
    | |_^
    |
    = note: `-D clippy::missing-const-for-fn` implied by `-D clippy::nursery`
    = help: for further information visit https://rust-lang.github.io/rust-clippy/master/index.html#missing_const_for_fn

error: aborting due to previous error

error: Could not compile `seed`.

To learn more, run the command again with --verbose.
[cargo-make] ERROR - Error while executing command, exit code: 101
[cargo-make] WARN - Build Failed.

```
which I don't think it's realted to this PR

Reply:
> which I don't think it's realted to this PR

Unfortunately we cannot merge PRs with failed tests, so please fix it. Similar errors are in most cases caused by updated Rust compiler or Clippy and can be easily fixed.

Reply:
Looks like this error disappear when I compile with stable rust 1.38, should I switch to nighliy and fix them or there is no need ?

Reply:
Latest Stable has the main priority,  but it would be nice to make it compatible with Nightly because we are using it for debugging. Thanks.

Reply:
I believe these Clippy warning is not related to this PR at all, so I won't fix them.